src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -35,22 +35,22 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
   case class SFNRequest(stateMachineArn: String, name: String, input: String)
 
   val reference = "TEST-REFERENCE"
-  val uuids: List[String] = List(
-    "c7e6b27f-5778-4da8-9b83-1b64bbccbd03",
-    "61ac0166-ccdf-48c4-800f-29e5fba2efda",
-    "4e6bac50-d80a-4c68-bd92-772ac9701f14",
-    "c2e7866e-5e94-4b4e-a49f-043ad937c18a",
-    "27a9a6bb-a023-4cab-8592-39b44761a30a"
+  val uuids: List[(String, String)] = List(
```

Comment:
Do you think it's worth adding what that second string is, in the val name "uuidsAndX" because it's hard to tell what it is?

Reply:
Good point, it's uuids and checksums. It doesn't look like a checksum because S3 does weird things to it which we have to replicate here. I've changed the name.